provisioning/bmo_pod.go

```diff
@@ -209,9 +209,14 @@ func newBMOPodTemplateSpec(info *ProvisioningInfo, labels *map[string]string) (*
 
 	containers := injectProxyAndCA([]corev1.Container{container}, info.Proxy)
 
+	podAnnotations := map[string]string{
```

Comment:
Could you copy over `podTemplateAnnotations` and update the copy with the additional annotation? This way we won't forget to update BMO if podTemplateAnnotations ever gets updated.